apps/event-system/services/stripe/stripe-webhook.service.ts

```diff
@@ -65,6 +65,7 @@ export default {
 
           if (key && subscriptionCreated?.status === 'active') {
             const billing = {
+              throughput: parseInt(process.env.DEFAULT_CLIENT_THROUGHPUT) || 500,
```

Comment:
@peterkuczera DEFAULT_CLIENT_THROUGHPUT environment needs to be added in the typescript services with a value of 500

Reply:
It's been added in https://github.com/integration-os/infrastructure/pull/455